### PR TITLE
Adjust tests for rhel-8-3

### PR DESCRIPTION
**Pull request by marusak**
Blocked on https://github.com/cockpit-project/bots/pull/776

[1](https://logs.cockpit-project.org/logs/pull-13928-20200420-104709-0964f5ea-rhel-8-3-bots-776/log.html)

### CODE REVIEW

File: test/verify/check-journal [hunk 1]
```diff
@@ -414,7 +415,8 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
         b.wait_present(sel)
 
     @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable",
-               "ubuntu-1804", "ubuntu-2004", "fedora-coreos", "rhel-8-2", "rhel-8-2-distropkg", "centos-8-stream")
+               "ubuntu-1804", "ubuntu-2004", "fedora-coreos", "rhel-8-2", "rhel-8-2-distropkg",
+               "centos-8-stream", "rhel-8-3")
```
Comment (martinpitt) [inline test/verify/check-journal:419]: Nitpick: Would be nicer to keep all the RHELs together.

**Comment by marusak** [comment]
[testVdo](https://logs.cockpit-project.org/logs/pull-13928-20200421-060603-5ec537d9-rhel-8-3-bots-776/log.html#166) seems to be stuck on:
```
Error creating file system: Command-line `wipefs --all "/dev/dm-3"' exited with non-zero exit status 1: wipefs: error: /dev/dm-3: probing initialization failed: Device or resource busy
```
I'll spin it once more as I cannot reproduce this locally. (Also it is destructive tests, so previous tests should not affect it).

**Comment by marusak** [comment]
Green!

File: test/verify/check-journal (hunk 1 above)
Comment (marusak) [reply test/verify/check-journal]: Done

File: test/verify/check-packagekit [hunk 2]
```diff
@@ -44,7 +44,7 @@ class NoSubManCase(PackageCase):
 
         # Disable Subscription Manager on RHEL for these tests; subscriptions are tested in a separate class
         # On other OSes (Fedora/CentOS) we expect sub-man to be disabled in yum, so it should not get in the way there
-        if self.machine.image.startswith("rhel") or self.machine.image.startswith("centos-8"):
+        if self.machine.image in ["rhel-8-2", "rhel-8-2-distropkg", "centos-8-stream"]:
```
Comment (martinpitt) [inline test/verify/check-packagekit:47]: Fun, earlier this morning I saw exactly that failure in an 8.3 downstream test. Is rhsmd just not a thing any more?

Comment (marusak) [reply test/verify/check-packagekit:47]: > Is rhsmd just not a thing any more?

indeed, it is present only when you install `subscription-manager-gui`.

**Pull request merged**